src/Raven.Client/Util/StreamWithTimeout.cs

```diff
@@ -139,13 +149,6 @@ public override int Read(byte[] buffer, int offset, int count)
 
         public override async Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken)
         {
-            if (_canBaseStreamTimeoutOnRead)
```

Comment:
let's remove the `async/await` from here as well

Reply:
done